tools/include/compiler_options.hpp.in

```diff
@@ -546,14 +547,31 @@ static void GetLdDefaults(std::vector<std::string>& ldopts) {
    } else {
 #ifdef __APPLE__
       ldopts.insert(ldopts.end(), {"-arch", "x86_64", "-macosx_version_min", "10.13", "-framework", "Foundation", "-framework", "System"});
+
+      llvm::SmallString<256> Output;
```

Comment:
please keep naming according to the file. Rest of file has snake case

Reply:
good call; it's a tough mix because LLVM's naming standard is different than ours..